src/components/EnterpriseApp/data/enterpriseCurationReducer.js

```diff
@@ -54,12 +71,66 @@ function getHighlightSetsFromState(state) {
   return state.enterpriseCuration?.highlightSets || [];
 }
 
+/**
+ * Helper function to determine if a content is archived.
+ * If the length > 1 for the course run status e.g. ["archived", "published"]
+ * the content is not considered archived. An "unpublished" course run is not considered "archived" because
+ * it could have a scheduled course run in the future.
```

Comment:
[clarification] I suppose the reason I'm concerned about `unpublished` is that a course that might have `courseRunStatuses:  ["archived", "unpublished"]`, the unpublished course run is effectively unusable by learners (hidden). The only _actionable_ course run status in the `courseRunStatuses` array remaining that's relevant / displayed to learners would be `archived`, despite having a `courseRunStatuses` length of 2.

With the logic, as is, in `isArchivedContent`, we would consider the course to be non-archived despite the only available course run displayed to learners being archived.